FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -29,14 +29,51 @@ public final class CoreDataFeedStore: FeedStore {
 	}
 
 	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+		let context = self.context
+		context.perform {
+			do {
+				let request = NSFetchRequest<ManagedCache>(entityName: "\(ManagedCache.self)")
+				request.returnsObjectsAsFaults = false
+				if let cache = try context.fetch(request).first {
```

Comment:
There's already a helper `ManagedCache.find`. No need to rebuild `NSFetchRequest`

Please remove fetch request and replace it with `ManagedCache.find`

```suggestion
				if let cache = try ManagedCache.find(in: context) {
```